crates/corro-api-types/src/lib.rs

```diff
@@ -35,6 +35,7 @@ pub enum TypedQueryEvent<T> {
     },
     Change(ChangeType, RowId, T, ChangeId),
     Error(CompactString),
+    Notify(ChangeType, T)
```

Comment:
I don't think we should reuse this enum. The updates manager is always going to be sending `Notify` and the subs manager will never be sending `Notify`. Re-using the same enum is more complicated for clients because now they have to handle an enum variant that'll never matcher anyway.

Reply:
I was thinking we might want to send `Columns` at the start of the updates but that could be put in the new enum too.

Reply:
Yes, but we can also assume clients know the primary keys of a table, I think. On subscriptions, there can be a lot of columns and they can have generated names, but not for these.